### PR TITLE
Added notes about Quick Installer deprecation in 3.9

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://trello.com/c/SnucbUnb/772-document-depreciate-announce-quick-installer-will-be-removed-from-ocp-310

Preview Builds:
http://file.rdu.redhat.com/~ahardin/03072018/deprecate-quick-install/install_config/install/quick_install.html#overview

http://file.rdu.redhat.com/~ahardin/03072018/deprecate-quick-install/install_config/install/planning.html#installation-methods

http://file.rdu.redhat.com/~ahardin/03072018/deprecate-quick-install/install_config/install/host_preparation.html#what-s-next

http://file.rdu.redhat.com/~ahardin/03072018/deprecate-quick-install/upgrading/automated_upgrades.html#overview

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@sdodson @adellape PTAL. Thanks!

File: install_config/install/advanced_install.adoc [hunk 1]
```diff
@@ -513,12 +508,12 @@ which may cause invalid configurations. Example usage:
 |`openshift_docker_options`
 a|This variable configures additional `docker` options within
 *_/etc/sysconfig/docker_*, such as options used in
-xref:../../install_config/install/host_preparation.adoc#managing-docker-container-logs[Managing Container Logs]. 
-Use `json-file` or `journald`. The default is `journald`. 
-Example usage: 
+xref:../../install_config/install/host_preparation.adoc#managing-docker-container-logs[Managing Container Logs].
+Use `json-file` or `journald`. The default is `journald`.
```
[the remark this answers is not in the document]
Comment (ahardin-rh) [reply install_config/install/advanced_install.adoc]: @vikram-redhat Sure. Thanks for letting me know.  I can address that in a separate PR as @adellape and I collaborate on updates.

**Comment by ahardin-rh** [comment]
@jiajliu PTAL. Thanks!

**Review by sdodson — APPROVED** [review]

**Comment by ahardin-rh** [comment]
There is a collective, end-to-end QE review happening for all 3.9 Installation/Upgrade tasks, including this one. Merging this and cherry-picking so that the changes are part of the 3.9 preview build for their review.

**Pull request merged**